src/core/Global.ttslua

```diff
@@ -225,7 +225,11 @@ function onObjectEnterZone(zone, object)
   elseif zone.type == "Hand" and object.type == "Card" then
     -- make sure the card is face-up
     if object.is_face_down then
-      object.flip()
+      if object.held_by_color then
+        object.held_flip_index = 0
+      else
+        object.flip()
+      end
```

Comment:
Cards moved to the hand still flip, but if they are held by a player, they keep it held instead of losing control.

Reply:
Not sure about this change. Did some testing and dragging a card to hand doesn't flip the card when I was hoping/expecting it. What's the case where this is an improvement?

Reply:
I don't understand? It's still just flipping the card, just that you don't lose control

Reply:
I'll try to make a video later today/tomorrow and send it to you, but it's only flipping about 75%-80% of the time.